packages/terra-visually-hidden-text/src/VisuallyHiddenText.jsx

```diff
@@ -6,6 +6,10 @@ import styles from './VisuallyHiddenText.module.scss';
 const cx = classNames.bind(styles);
 
 const propTypes = {
+  /**
+   * Callback ref to pass into the dom element.
+   */
```

Comment:
want to add how this is useful for aria-live - similar to the changelog entry

Reply:
Updated in this commit: https://github.com/cerner/terra-core/pull/2495/commits/63d75ab828f340f68ef3f820950f28e568fe5b6a